### DISCUSSION TITLE
Number of letters of recommendation

### DISCUSSION THREAD

**Discussion by anhtienng** [nguyenthanhvuh/phd-cs-us#15] [Q&A] [answered]
Hi Vu,

I am preparing the application for PhD and I have 4 LoRs.
On some universities websites, the suggested number of LoRs are 3, but I still can add more than 3 recommenders on the application portals. 
- Should I submit 4 letters in that case ? 
- Will it hurt/help my application with 4 letters ?
- Some universities said they only considered 3 letters although 4 letters are submitted. How will they consider 3 or 4 submitted letters ? Is that based on the submission time (3 earliest letters) ?

Thank you.

**Comment by nguyenthanhvuh** [comment, marked as answer]
just submit your strongest 3

**Reply by anhtienng** [reply]
I got it, thank you
